models/aeif_cond_beta_multisynapse.cpp

```diff
@@ -144,84 +153,75 @@ aeif_cond_beta_multisynapse::Parameters_::get( DictionaryDatum& d ) const
   def< double >( d, names::I_e, I_e );
   def< double >( d, names::V_peak, V_peak_ );
   def< double >( d, names::gsl_error_tol, gsl_error_tol );
-  def< int >( d, "n_synapses", num_of_receptors_ );
   def< bool >( d, names::has_connections, has_connections_ );
 }
 
 void
 aeif_cond_beta_multisynapse::Parameters_::set( const DictionaryDatum& d )
 {
-
   updateValue< double >( d, names::V_th, V_th );
   updateValue< double >( d, names::V_peak, V_peak_ );
   updateValue< double >( d, names::t_ref, t_ref_ );
   updateValue< double >( d, names::E_L, E_L );
   updateValue< double >( d, names::V_reset, V_reset_ );
-  updateValue< double >( d, names::E_ex, E_ex );
-  updateValue< double >( d, names::E_in, E_in );
 
   updateValue< double >( d, names::C_m, C_m );
   updateValue< double >( d, names::g_L, g_L );
 
-  std::vector< double > tau_tmp;
-  if ( updateValue< std::vector< double > >( d, names::taus_decay, tau_tmp ) )
-  {
-    if ( tau_tmp.size() < taus_decay.size() && has_connections_ == true )
+  std::vector< double > Erev_tmp, taur_tmp, taud_tmp;
+  bool Erev_flag =
+    updateValue< std::vector< double > >( d, names::E_rev, Erev_tmp );
+  bool taur_flag =
+    updateValue< std::vector< double > >( d, names::taus_rise, taur_tmp );
+  bool taud_flag =
+    updateValue< std::vector< double > >( d, names::taus_decay, taud_tmp );
+  if ( Erev_flag || taur_flag || taud_flag )
+  { // receptor arrays have been modified
+    if ( !Erev_flag || !taur_flag || !taud_flag )
     {
       throw BadProperty(
-        "The neuron has connections, therefore the number of ports cannot be "
-        "reduced." );
+        "The reversal potential, synaptic rise time and synaptic decay time "
+        "arrays must be modified altogether." );
```

Comment:
It's a problem with my English, I will fix the error message.... About the implementation of the constraints, I followed @heplesser suggestions in comment https://github.com/nest/nest-simulator/issues/510#issuecomment-252402610 when he says:
"if a call to SetStatus contains taus_syn, it also must contain E_rev, and they must be of the same length". In case of aeif_cond_alpha_multisynapse there is no taus_syn, but taus_rise and taus_decay, so I guess that SetStatus should either contain none of them or all three...
If I understand well, what you are proposing is that you can also call SetStatus with one or two of them as far as they have the same length that they had previously. For me it would be fine. @heplesser what do you think about this?